packages/loader/driver-utils/src/runWithRetry.ts

```diff
@@ -80,13 +80,20 @@ export async function runWithRetry<T>(
 						retry: numRetries,
 						duration: performance.now() - startTime,
 						fetchCallName,
+						// TODO: Remove when typescript version of the repo contains the AbortSignal.reason property
```

Comment:
I'd suggest opening an ADO task and referencing ADO # in the code - it would be extremely hard to find all these cases otherwise later.